example/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ find_package(Threads)
 
 # --- find our library
 
-find_package(SC REQUIRED)
+find_package(SC QUIET)
```

Comment:
If we don't expect to find this, would everything work by removing this line entirely?